.github/CODEOWNERS

```diff
@@ -1,5 +0,0 @@
-# See for instructions on this file https://help.github.com/articles/about-codeowners/
```

Comment:
Please remove codeowners from the PR, we still need it :)

Reply:
I believe this is done, can you rereview @daltondhcp

Reply:
Both of the files are removed entirely in your branch, meaning that they would be removed in upstream at merge, which we obviously don't want. Please restore CODEOWNERS and .gitignore in your branch from Azure/AzOps:main and push. 
![image](https://user-images.githubusercontent.com/16622613/147044553-4898818e-bd94-4664-8d51-0488e3c2d172.png)